src/UI/DataHTMLTooltip.js

```diff
@@ -50,6 +51,17 @@ ${css}
         this.autofillButtons.forEach((btn) => {
             this.registerClickableButton(btn, () => {
                 callbacks.onSelect(btn.id)
+
```

Comment:
We should probably put this together with the other pixel [here](https://github.com/duckduckgo/duckduckgo-autofill/blob/3a329ee47224295fd4e8e85d8674daa126145134/src/DeviceInterface/InterfacePrototype.js#L471). You still have access to the `id` there so it should be easy to move this there.